tapestry.js

```diff
@@ -2140,6 +2163,23 @@ function exitViewMode() {
     startForce();
 }
 
+function createLightBoxFooter(description, mediaType) {
+    if (description) {
+        var descriptionContent = document.createElement("div");
```

Comment:
The div was needed because i added a horizontal line for text nodes where video nodes won't have it. but since we're moving this to the sidebar, this function will be changed.